zerver/views/auth.py

```diff
@@ -423,6 +423,7 @@ def remote_user_jwt(request: HttpRequest) -> HttpResponse:
     try:
         realm = get_realm(subdomain)
     except Realm.DoesNotExist:
+        logging.warning("Wrong subdomain")
```

Comment:
I don't think we need this. If there wasn't any logging earlier we probably are not interested in logging this. Can you tell why you've added this? (I'm just curious if it's something we need to do)

Reply:
The `mock.patch('logging.warning')` object was there but no logs getting generated and tested, so I added the log statement. Should I remove the log and test statement both?

Reply:
Yeah, I think it's fine to remove that `mock.patch('logging.warning')` as it doesn't generate any logs. It think it'd clear to keep that in a separate commit.

Reply:
made this change in [commit](https://github.com/zulip/zulip/pull/15708/commits/b4970ca1a1c91d2cb14b6ef50c38ca7b0dc9b067)